### ISSUE TITLE
Enforce/Check capitalization properly (False positives for 'German' and 'Russian')

### ISSUE THREAD

**Issue by luzpaz** [codespell-project/codespell#200]
Recently the following entries were added to the `dictionary.txt`
`german->German` [[link](https://github.com/lucasdemarchi/codespell/commit/00955e924ca6a1be486cf4905ecc105b77d577d6#diff-3bddc99f1f6dffc98ea11be5a5a31ae9R2188)]
`russian->Russian` [[link](https://github.com/lucasdemarchi/codespell/commit/00955e924ca6a1be486cf4905ecc105b77d577d6#diff-3bddc99f1f6dffc98ea11be5a5a31ae9R4248)]

Now when codespell encounters `German` it still reports it as incorrect:
```
./windows/netgen.rc:16: German  ==> German
./windows/netgen.rc:95: German  ==> German
```

```
// German (Germany) resources
	German  ==> German (Y/n) 
```

Same for `Russian`

**Comment by luzpaz** [comment]
@larsoner can you confirm ?

**Comment by larsoner** [comment]
we should probably get rid of those, I don't think we enforce/check capitalization properly

**Comment by luzpaz** [comment]
@larsoner i'll submit a PR once the open PR queue shrinks again

**Comment by luzpaz** [comment]
Related to https://github.com/lucasdemarchi/codespell/issues/74 ?

**Comment by peternewman** [comment]
Fixed with `sed -i -re "/^([^\-]+)->\1$/Id" dictionary.txt`

**Comment by buhtz** [comment]
Can't this be closed because of #298 is merged?
